cli/ceph/orch/orch.py

```diff
@@ -0,0 +1,132 @@
+from cli import Cli
+from cli.utilities.utils import config_dict_to_string, get_nodes_by_ids
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class OrchApplyServiceFailure(Exception):
+    pass
+
+
+class Orch(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(Orch, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} orch"
+
+    def ls(self, **kw):
+        """
+        Lists the status of the given services running in the Ceph cluster.
+
+        Args:
+            kw (Dict): Key/value pairs that needs to be provided to the installer.
+            Example:
+                Supported keys:
+                    service_type(str): type of service(mon, osd, mgr, mds, rgw)(Optional).
+                    service_name(str): name of host(Optional).
+                    export(bool): Whether to export the service specifications knows to
+                                  the orchestrator(Optional).
+                    format(str): the type to be formatted(yaml)(Optional).
+                    refresh(bool): Whether to refresh or not(Optional).
+        """
+        args = ""
+        for k, v in kw.items():
```

Comment:
This loop is going to be used in lot of places, please have separate method `build_cmd_args` in `cli/utilities/utils.py`.